modules/commons/src/main/java/org/apache/synapse/commons/resolvers/FilePropertyResolver.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.synapse.commons.resolvers;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.commons.util.FilePropertyLoader;
+
+/**
+ *  File Property resolver can be used to resolve file property variables in the synapse config.
+ */
+public class FilePropertyResolver implements Resolver {
+
+    private static final Log log = LogFactory.getLog(FilePropertyResolver.class);
+
+    private String input;
```

Comment:
Let's describe what the input is

Reply:
Fix with the commit https://github.com/wso2/wso2-synapse/commit/4d872fcac4d954fa4b59405b1dd501a8b792dd1f